client/utils/events.ts

```diff
@@ -419,9 +419,9 @@ const getDateStringForEvent = (event, dateOnly = false, useLocal = true, withTim
     // !! Note - expects event dates as instance of moment() !! //
```

Comment:
It might also help to add types to the signature, such as `eventSchedule: IEventItem['dates']` so it is clear what this function is expecting